cmd/build.go

```diff
@@ -324,10 +326,10 @@ func buildVerificationConfig(pubKey, pubKeyID, alg, scope string, excludeFiles [
 	return bundle.NewVerificationConfig(map[string]*keys.Config{pubKeyID: keyConfig}, pubKeyID, scope, excludeFiles), nil
 }
 
-func buildSigningConfig(key, alg, claimsFile string) *bundle.SigningConfig {
+func buildSigningConfig(key, alg, claimsFile, plugin string) *bundle.SigningConfig {
 	if key == "" {
 		return nil
 	}
 
-	return bundle.NewSigningConfig(key, alg, claimsFile)
+	return bundle.NewSigningConfig(key, alg, claimsFile).WithPlugin(plugin)
```

Comment:
What about the verification side of things ? 

Users should be able to specify the verification plugin on the command-line as well.  I would imagine the plugin provided on the command-line would take precedence over the one provided in the signatures config. WDYT ?

Reply:
I'm thinking that in the vast majority of use cases, the verification plugin needed is always going to depend on the signing plugin used, so it makes sense to just register them under the same key.  My recommendation would be to roll with this until someone brings a need to override the verification plugin on the CLI (just in case its YAGNI).

Reply:
We should maybe add a note about this in the docs so users are aware of this at least.